docs/version0.25.md

```diff
@@ -95,6 +96,11 @@ OpenJ9 on AIX&reg; systems uses the hardware-accelerated `zlibNX` if the Nest ac
 
 The `%sysname` dump token is added on z/OS, which equates to the SYSNAME sysparm. See [Dump agent tokens](xdump.md#dump-agent-tokens).
 
+### Single build for compressed references and non-compressed references
+
+A single (mixedrefs) build now supports both compressed references and non-compressed references. The object reference mode is selected at runtime based on the specified heap size ([`-Xmx`](#xms.md)), or via command line options that control compressed references behavior. The `-Xnocompressedrefs` command line option must be used on a mixedrefs build when non-compressed references is desired for smaller heap sizes that otherwise select compressed references automatically. This is a change in behavior from using a large heap build. The `compressedrefs` directory is not present in a mixedrefs build.
```

Comment:
A suggested rewording for:
> The `-Xnocompressedrefs` command line option must be used on a mixedrefs build when non-compressed references is desired for smaller heap sizes that otherwise select compressed references automatically

Perhaps something like:
> If non-compressed references is desired when using a mixedrefs build (i.e. for smaller heap sizes that select compressed references automatically), the command line option `-Xnocompressedrefs` must be used.

Reply:
Thanks. Updated as suggested.